src/main/java/org/seqdoop/hadoop_bam/VCFRecordReader.java

```diff
@@ -140,7 +174,26 @@
 				}
 			}
 
-			final VariantContext v = codec.decode(line);
+                        final VariantContext v;
+                        try {
+				v = codec.decode(line);
+			} catch (TribbleException e) {
+				if (stringency == ValidationStringency.STRICT) {
+					if (stringency == ValidationStringency.LENIENT &&
```

Comment:
If strict, we're throwing `e`, so why log here?  Shouldn't that be downstream's responsibility?

Reply:
htsjdk doesn't include the offending line in all exception messages IIRC; logging an error message with the line and exception makes debugging the exception much easier.